src/rulesets/rest/2022-05-25/utils.ts

```diff
@@ -10,8 +10,14 @@ export const isBatchPostOperation = (requests) => {
   const request = requests.find(
     (request) => request.contentType === "application/vnd.api+json",
   );
-
-  return request ? request.value.flatSchema.type === "array" : false;
+  const requestSchema = request?.raw.schema;
```

Comment:
@niclim Question about `flatSchema` vs `raw` here... I found `raw` in the debugger, is this the correct way to get access to the OpenAPI document object model? `flatSchema` seemed to drop the properties in the top-level schema object so I couldn't use that. Just want to make sure I'm building on solid ground here. Thanks!

Reply:
Oh I see, yeah this is fine. Flatschema is a flattened version of the openapi spec and since you are looking at this from the request level this is ok to do

Reply:
Thanks!